docs/source/modules/datasets.rst

```diff
@@ -86,111 +86,390 @@ of vocabs.
    * - Name
      - size
      - characters
-   * - ascii_letters
-     - 52
-     - abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ
-   * - currency
+   * - arabic_diacritics
+     - 8
+     - ًٌٍَُِّْ
+   * - arabic_digits
+     - 10
+     - ٠١٢٣٤٥٦٧٨٩
+   * - arabic_letters
+     - 37
+     - ءآأؤإئابةتثجحخدذرزسشصضطظعغـفقكلمنهوىي
+   * - arabic_punctuation
      - 5
-     - £€¥¢฿
+     - ؟؛«»—
+   * - persian_letters
+     - 5
+     - پچڢڤگ
+   * - bangla_digits
+     - 10
+     - ০১২৩৪৫৬৭৮৯
+   * - bangla_letters
+     - 69
+     - অআইঈউঊঋঌএঐওঔকখগঘঙচছজঝঞটঠডঢণতথদধনপফবভমযরলশষসড়ঢ়য়ৠৡৰৱৼহ়ঽািীুূৃেৈোৌ্ৎংঃঁ
+   * - generic_cyrillic_letters
+     - 58
+     - абвгдежзийклмнопрстуфхцчшщьюяАБВГДЕЖЗИЙКЛМНОПРСТУФХЦЧШЩЬЮЯ
+   * - russian_cyrillic_letters
+     - 6
+     - ёыэЁЫЭ
+   * - russian_signs
+     - 2
+     - ъЪ
+   * - ancient_greek
+     - 49
+     - αβγδεζηθικλμνξοπρστςυφχψωΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩ
+   * - gujarati_consonants
+     - 34
+     - કખગઘઙચછજઝઞટઠડઢણતથદધનપફબભમયરલળવશષસહ
+   * - gujarati_digits
+     - 10
+     - ૦૧૨૩૪૫૬૭૮૯
+   * - gujarati_punctuation
+     - 5
+     - ૐ૱૰ઽ॥
+   * - gujarati_vowels
+     - 14
+     - અઆઇઈઉઊઋઌઍએઐઑઓઔ
+   * - gujarati_diacritics
+     - 26
+     - ઁંઃ઼ાિીુૂૃૄૅેૈૉોૌ્ૢૣૺૻૼ૽૾૿
+   * - hindi_digits
+     - 10
+     - ०१२३४५६७८९
+   * - hindi_letters
+     - 49
+     - अआइईउऊऋॠऌॡएऐओऔंःकखगघङचछजझञटठडढणतथदधनपफबभमयरलवशषसह
+   * - hindi_punctuation
+     - 9
+     - ।,?!:्ॐ॰॥
+   * - hebrew_cantillations
+     - 31
+     - ֑֖֛֢֣֤֥֦֧֪֚֭֮֒֓֔֕֗֘֙֜֝֞֟֠֡֨֩֫֬֯
+   * - hebrew_letters
+     - 27
+     - אבגדהוזחטיךכלםמןנסעףפץצקרשת
+   * - hebrew_specials
+     - 50
+     - ׯװױײיִﬞײַﬠﬡﬢﬣﬤﬥﬦﬧﬨ﬩שׁשׂשּׁשּׂאַאָאּבּגּדּהּוּזּטּיּךּכּלּמּנּסּףּפּצּקּרּשּתּוֹבֿכֿפֿﭏ
+   * - hebrew_punctuation
+     - 12
+     - ֽ־ֿ׀ׁׂ׃ׅׄ׆׳״
+   * - hebrew_vowels
+     - 13
+     - ְֱֲֳִֵֶַָׇֹֺֻ
    * - digits
      - 10
      - 0123456789
+   * - ascii_letters
+     - 52
+     - abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ
    * - punctuation
      - 32
-     - !"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~
+     - !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
+   * - currency
+     - 5
+     - £€¥¢฿
+   * - latin
+     - 94
+     - 0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
+   * - english
+     - 100
+     - 0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~°£€¥¢฿
+   * - albanian
+     - 104
+     - 0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~°£€¥¢฿çëÇË
+   * - afrikaans
+     - 114
+     - 0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~°£€¥¢฿èëïîôûêÈËÏÎÔÛÊ
+   * - azerbaijani
+     - 111
+     - 0123456789abcdefghijklmnopqrstuvxyzABCDEFGHIJKLMNOPQRSTUVXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~°£€¥¢฿çəğöşüÇƏĞÖŞÜ₼
+   * - basque
+     - 104
+     - 0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~°£€¥¢฿ñçÑÇ
+   * - bosanski
```

Comment:
I noticed this is the only language whose name is not in English.

Reply:
Ah good catch correct it should be bosnian ^^ will update this :+1:

Reply:
Corrected :+1: